terraform/cleanup.sh

```diff
@@ -137,7 +137,7 @@ gcloud scheduler jobs delete nightly-builds \
     || true
 
 # Remove existing Terraform state (Part 1)
-pushd "terraform/ops"
+pushd terraform/environments/ops
 terraform init
 terraform apply \
```

Comment:
future change: could we switch to `terraform destroy` for brevity and clarity? This through me a bit as I was trying to figure out why a cleanup script was creating resources.